dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-zeppelin.ts

```diff
@@ -45,16 +45,16 @@ export function useZeppelin(model: { [field: string]: any }): IJsonItem[] {
       name: t('project.node.zeppelin_paragraph_id'),
       props: {
         placeholder: t('project.node.zeppelin_paragraph_id_tips')
-      },
-      validate: {
-        trigger: ['input', 'blur'],
-        required: true,
-        validator(validate: any, value: string) {
-          if (!value) {
-            return new Error(t('project.node.zeppelin_paragraph_id_tips'))
-          }
-        }
       }
+//       validate: {
```

Comment:
I think it should be removed if unuse, WDYT? @songjianet @Amy0104

Reply:
Sure, will remove it.